packages/tfjs-node-helpers/src/testing/metrics/roc-auc.ts

```diff
@@ -0,0 +1,40 @@
+import { binarize } from '../../utils/binarize';
+import { Metric } from '../metric';
+import { MetricCalculator, MetricCalculatorData } from '../metric-calculator';
+import { FPRMetricCalculator } from './fpr';
+import { RecallMetricCalculator } from './recall';
+
+export class ROCAUCMetricCalculator extends MetricCalculator {
+  constructor(private thresholdNumber: number = 21) {
+    super();
+  }
+
+  public calculate({ trueValues, probabilities }: MetricCalculatorData): Metric {
+    let area = 0;
+    let threshold = 0;
+    const step = 1 / (this.thresholdNumber - 1);
```

Comment:
@VladimirGlinskii Let's rename this to `stepSize`. Also, why do we subtract `1` from `this.thresholdNumber` here?